src/plugin.py

```diff
@@ -226,6 +262,26 @@ def _force_utc(self):
             logging.exception("Can not parse backend response")
             raise UnknownBackendResponse()
 
+        try:
```

Comment:
Is parsing local files the only solution for getting game times for family shared games? I guess its ok if there is no other way but i want to be sure we checked other possibilities first

Reply:
Can't find other way sorry